Analytics-CSharp/Segment/Analytics/Analytics.cs

```diff
@@ -82,18 +84,6 @@ public void Process(RawEvent incomingEvent)
         /// <returns>Anonymous Id</returns>
         public string AnonymousId()
```

Comment:
since we already introduce breaking changes, i.e. removed `AnonymousIdAsync`, we can simplified this to:
```
public string AnouymousId => userInfo.AnouymousId;
```